### PR TITLE
Fix some pipeline errors

### PR DESCRIPTION
**Pull request by snnn**

### CODE REVIEW

File: tools/ci_build/github/azure-pipelines/templates/packaging-pipeline-steps.yml [hunk 1]
```diff
@@ -80,19 +80,20 @@ steps:
     workingDirectory: $(Build.SourcesDirectory)
   displayName: "docker run build.py --wheel_file /torch_ort_src/dist/$whlfilename"
 
-- task: AzureCLI@2
-  condition: and(succeeded(), eq(variables.UploadWheel, 'yes'), eq(variables.BuildType, 'nightly'))
```
Comment (snnn) [inline tools/ci_build/github/azure-pipelines/templates/packaging-pipeline-steps.yml:84]: Here UploadWheel is a parameter, not a variable. `variables.UploadWheel` would be always null. I wonder how it worked before.

Comment (snnn) [reply tools/ci_build/github/azure-pipelines/templates/packaging-pipeline-steps.yml:84]: Probably before #173 , the pipeline had the two variables.

**Review by baijumeswani — APPROVED** [review]

**Comment by baijumeswani** [comment]
Looks like I need to renew my access to this repo. Sorry, cannot give you an admin review rn.

**Comment by snnn** [comment]
I can merge it now.

**Pull request merged**